app/src/main/java/com/github/wnder/NetworkInformation.java

```diff
@@ -0,0 +1,20 @@
+package com.github.wnder;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import static androidx.core.content.ContextCompat.getSystemService;
+
+public class NetworkInformation {
```

Comment:
Once again making a private constructor might be a good idea just to make it clear it's not meant to be intanciated.